packages/contact-center/cc-components/src/components/task/CallControl/CallControlCustom/consult-transfer-popover.tsx

```diff
@@ -84,19 +89,26 @@ const ConsultTransferPopoverComponent: React.FC<ConsultTransferPopoverComponentP
     </ListNext>
   );
 
-  const noQueues = queuesData.length === 0;
+  const noQueues = !allowConsultToQueue || queuesData.length === 0;
   const noDialNumbers = !showDialNumberTab || dialNumbers.length === 0;
-  const noEntryPoints = !showEntryPointTab || entryPoints.length === 0;
+  const noEntryPoints = !isEntryPointTabVisible || entryPoints.length === 0;
 
-  const hasAnyData = !noAgents || !noQueues || !noDialNumbers || !noEntryPoints;
+  const consultTransferManualAction = buildConsultTransferQuickAction(
```

Comment:
This seems alright except the name `buildConsultTransferQuickAction`. It sounds like the method would give back a React Component while all that it does is to tell whether the quick action needs to be shown or not.